ivy/stateful/converters.py

```diff
@@ -243,10 +177,12 @@ def __init__(self, *args, native_module, device, devices, **kwargs):
             def _create_variables(self, device, dtype):
                 return self._hk_params
 
-            def _build(self, *args, **kwargs):
+            def _build(self, params_hk, *args, **kwargs):
                 args, kwargs = ivy.args_to_native(*args, **kwargs)
                 # noinspection PyUnresolvedReferences
-                params_hk = self._native_module.init(RNG, *args, **kwargs)
+                # params_hk = self._native_module.init(RNG, *args, **kwargs)
+
+                # TODO: this _hk_flat_map_to_dict does not seem to be working fine ig ?
```

Comment:
should we keep this as it is now? isn't the `RNG` line needed?

Reply:
since we are passing the params explicitly, we don't need it anymore (RNG is mostly for params init)